### PR TITLE
I have done my homework 1

### PR DESCRIPTION
**Pull request by Denis-Pikus**
Homework1

### CODE REVIEW

**Review by olgaZskiba — APPROVED** [review]

File: IntensiveJavaGuru/pom.xml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example.dpdev</groupId>
+    <artifactId>IntensiveJavaGuru</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <sonar.projectKey>javaGuruBY_java-free-stream-5</sonar.projectKey>
```
Comment (zhdanok) [inline IntensiveJavaGuru/pom.xml:12]: Did  you create your own accaunt on sonarcloud.io?

File: IntensiveJavaGuru/pom.xml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                        <id>jacoco-check</id>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <rule>
+                                    <limits>
+                                        <limit>
+                                            <minimum>0.0</minimum>
```
Comment (zhdanok) [inline IntensiveJavaGuru/pom.xml:82]: 0.5 for homework

File: IntensiveJavaGuru/src/main/java/dpdev/homework_1/operations/Calculator.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    }
+
+    public static double divide(int a, int b) {
+        if (a == 0) {
+            throw new ArithmeticException("Divide by zero!");
+        } else {
+            return (double) a / b;
+        }
+    }
+
```
Comment (zhdanok) [inline IntensiveJavaGuru/src/main/java/dpdev/homework_1/operations/Calculator.java:26]: add  operation "reminder" -  a%b

File: IntensiveJavaGuru/src/test/java/dpdev/homework_1/operations/CalculatorTest.java [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+import org.slf4j.LoggerFactory;
+
+public class CalculatorTest extends TestCase {
+    static Logger logger = LoggerFactory.getLogger(CalculatorTest.class);
+
+    @Test
+    public void testAddition() {
+        int a = 5;
+        int b = 3;
+        int expectedResult = a + b;
```
Comment (zhdanok) [inline IntensiveJavaGuru/src/test/java/dpdev/homework_1/operations/CalculatorTest.java:15]: As I know expectedResult should be a number

File: IntensiveJavaGuru/src/test/java/dpdev/homework_1/operations/CalculatorTest.java [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        int actualResult = Calculator.addition(a, b);
+        assertEquals(expectedResult, actualResult);
+        logger.info("{} + {} = {}", a, b, a + b);
+    }
+
+    @Test
+    public void testSubtraction() {
+        int a = 5;
+        int b = 3;
+        int expectedResult = a - b;
```
Comment (zhdanok) [inline IntensiveJavaGuru/src/test/java/dpdev/homework_1/operations/CalculatorTest.java:25]: see above

**Review by zhdanok — APPROVED** [review]
see notes

**Review by avp02 — APPROVED** [review]
lgfm

**Review by integratorBY — COMMENTED** [review]
lgfm

**Pull request merged**